test-data/unit/check-incremental.test

```diff
@@ -1,21 +1,25 @@
 -- Checks for incremental mode (see testcheck.py).
 -- Each test is run twice, once with a cold cache, once with a warm cache.
--- Before the tests are run the second time, any *.py.next files are copied to *.py.
+-- Before the tests are run the second time, any *.py.2 files are copied to *.py
+-- (and *.py.3 files for the third step, and so on).
```

Comment:
A little awkwards wording, perhaps "Before the tests are run again, in step N any *.py.N files are ..." ? You might not need to "and *.py.3 etc." part then.